### ISSUE TITLE
(Beta Build 4) Cannot open any projects made with prior versions

### ISSUE THREAD

**Issue by nullbunny** [YoYoGames/GameMaker-Bugs#4670]
### Description
I submitted this bug with Beta 3 before - I wasn't sure if this was supposed to be fixed yet so I figured I'd submit a fresh one.

### Steps To Reproduce
Trying to open any project made with a prior version fails.

### Which version of GameMaker are you reporting this issue for?
IDE v2024.200.0.486 Runtime v2024.200.0.502

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

[547c8a7a-c3d3-415f-a67b-6b19cda6ca1c](https://api.gamemaker.io/api/github/downloads/547c8a7a-c3d3-415f-a67b-6b19cda6ca1c)

**Comment by nullbunny** [comment]
I also had this error pop up the first time I launched it. I don't know if it is related - if not, I can submit this as its own bug.

![2024-02-08 18_00_59-Start Page - GameMaker-Beta](https://github.com/YoYoGames/GameMaker-Bugs/assets/5711400/fc223e4d-778b-415f-ad4a-cc93357e21c8)

**Comment by YYDan** [comment]
In general, btw, you don't ever need to duplicate your own reports - you should be able to see if your earlier bug is still open or has been verified as fixed.

Also, there's no project in your package, so there's nothing we can do for this one just now.

Could you file a new private bug report and allow the bug-reporter to send us that project/go .zip up that project yourself and add it as an attachment, please? Then we can link the new package to this report.

**Comment by nullbunny** [comment]
Gotcha, thanks.

I attached a sample project to this message:

[LayerTest.zip](https://github.com/YoYoGames/GameMaker-Bugs/files/14212588/LayerTest.zip)

**Comment by gnysek** [comment]
If you opened that project in previous betas and now opened it again in latest beta, it might be why. You should try to open project created between 2.0 and 2022.11 (or LTS) to test it properly, as things may break between beta builds.

**Comment by nullbunny** [comment]
I also cannot open my main project, which is in source control - opening the project didn't change any of the files. I also tried to jump back a few weeks worth of commits and repeated trying to open it to no avail.

Another test I did was creating a completely new project using the latest stable version. All I did was create a blank game project, create a single sprite, create a single object, assign sprite to object and put both instances of the object and the sprite (on an asset layer) in the room that gets created with the blank project.

After creating the project I zipped it up and named it "Created in latest stable".

I then tried to open the project in Beta 4 and got the error.

I zipped up the folder again and named it "After trying to open in Beta 4".

Going back to the latest stable, it opens without any issues.

I don't think any of the files got change, but I hope this helps finding the bug.

[TestForBeta4 (Created in latest stable).zip](https://github.com/YoYoGames/GameMaker-Bugs/files/14214021/TestForBeta4.Created.in.latest.stable.zip)

[TestForBeta4 (After trying to open in Beta 4).zip](https://github.com/YoYoGames/GameMaker-Bugs/files/14214022/TestForBeta4.After.trying.to.open.in.Beta.4.zip)
